client/src/store/new-request-module.ts

```diff
@@ -609,6 +608,12 @@ export class NewRequestModule extends VuexModule {
       homeJurisNum: this.corpNum
     }
   }
+  get currentIssue () {
+    if (this.analysisJSON && this.analysisJSON.issues && Array.isArray(this.analysisJSON.issues)) {
```

Comment:
Looks OK but a couple of comments:
- you can use TS conditional chaining here, eg. `if (this.analysisJSON?.issues ...`
- let's start adding return types to methods/getters, to make it easier to review and implement new code

Reply:
Merge fragment. Not sure why git detected that as a change.

Reply:
OK. This is another reason why a rebase would have been great before review :)